src/components/Chat.js

```diff
@@ -401,7 +441,7 @@ export default class Chat extends React.Component {
             {quickReplies.length > 0 ? <QuickReplies replies={quickReplies} onSelect={this._onSend} active={true} /> : null}
             <div style={{ float:"left", clear: "both", height: '0px', width: '0px', padding: '0px', margin: '0px', visibility: 'hidden' }} ref={(el) => { this.messagesEnd = el; }} />
           </MessageList>
-          <TextComposer onSend={this._onSend} onFocus={onFocus} onBlur={onBlur}>
+          <TextComposer onSend={this._onSend} onFocus={onFocus} onBlur={onBlur} iconFSData={iconFSData}>
```

Comment:
1. Whats `iconFSData` ?  what does FS stand for?
2. Why isn't it in `Chat`'s propTypes?  with documentation so that someone who uses this open-source library knows what it is?

Reply:
FS = fontawesome

Reply:
@TzelaDayan 
1. No one can figure that out. I see the same question @nahtnam asked in the widget repo
2. The interface you defined isn't FontAwesome specific is it?  its some SVG data... 
If this was FontAwesome specific all you'd need is a string like "fa-stroopwafel" to know which icon to show.
But we dont want `ebates-chat-kit` to k now FontAwesome.
3. I dont know where you got that SVG structure you're sending here but thats not an intuitive API to expose to the outside. Why not just let the user of the library decide what he wants to put there? let him specify an html, if he wants font awesome he can write `<span class="fas fa-stroopwafel"></span>`

Reply:
Since we are able to pass in any SVG we want, it is not limited to font awesome. Something more generalized like `sendIcon` or `sendSVGIcon` would be better.

Reply:
4. documentation. documentation. documentation :).  its an Open Source project...